node/src/proxy_client/stream_handler_pool.rs

```diff
@@ -333,6 +333,13 @@ impl StreamHandlerPoolReal {
         )
     }
 
+    fn filter_wildcard_ips(ip_addrs: Vec<IpAddr>) -> Vec<IpAddr> {
+        ip_addrs
+            .into_iter()
+            .filter(|&ip_addr| ip_addr != IpAddr::V4(Ipv4Addr::new(0, 0, 0, 0)))
```

Comment:
You could put this outside the loop, rather than constructing it every time through...